### ISSUE TITLE
"Pages" link to specific page Coach data doesn't have styles on Windows

### ISSUE THREAD

**Issue by rschoessler** [sitespeedio/sitespeed.io#1003]
```
Given I run a sitespeed test on a web page
And I am viewing the HTML results
When I go to the Pages link in the header menu
And I select a page URL to view the page details
Then I should see a formatted page with CSS styles
```
When I view on Mac, styling works fine.
When I view on Windows, the page with the "Coach" data shows no styles.

>file:///C:/sitespeed/sitespeed.io/sitespeed-result/myurl.com/2016-06-16-13-30-11/pages/myurl.com/index.html

![coach](https://cloud.githubusercontent.com/assets/3880296/16126780/9c2c2f84-33c7-11e6-90b3-1cb03e7f2cdc.png)

**Comment by soulgalore** [comment]
hey @rschoessler thanks for reporting, could you do a view source and just check what the link to the CSS looks like?

Best
Peter

**Comment by rschoessler** [comment]
So, if I edit index.html in 


Sure, here's what I see on Windows:

`<link rel="stylesheet" href="../css/index.min.css">`

On Mac I see:

`<link rel="stylesheet" href="../../css/index.min.css">`

if I update the .html manually on Windows to follow the same path as Mac, I get the styles.

Also, the Images on that page show the same issue and are fixed when updating the path.

**Comment by soulgalore** [comment]
I guess this could be a // on windows or something like that. Lets create storage manager tests, so we can verify that it really works.

**Comment by rschoessler** [comment]
Hi guys...it looks like the links in the header navigation on the .../pages/site_url/index.html are also broken.  They are looking for the pages in the /pages directory instead of the root folder for that particular run.

**Comment by soulgalore** [comment]
Yes, I think @tobli has a fix for this and soon he can commit it so you can test that it works.

**Comment by tobli** [comment]
Fixed via #1084
